product_expiration/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+Add product expiration date.
```

Comment:
Not technically true. It adds a field that contains a number of days after which the product expires, not a date.